client/my-sites/checkout/checkout-thank-you/marketplace/use-default-thank-you-footer.tsx

```diff
@@ -3,79 +3,122 @@ import { Button } from '@wordpress/components';
 import { useTranslate } from 'i18n-calypso';
 import { useCallback } from 'react';
 import { useSelector } from 'react-redux';
-import { ThankYouSectionProps } from 'calypso/components/thank-you/types';
+import { ThankYouSectionProps, ThankYouNextStepProps } from 'calypso/components/thank-you/types';
 import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 
-export function useDefaultThankYouFoooter( slugs: string[] ): ThankYouSectionProps {
+export function useThankYouFoooter(
+	pluginSlugs: Array< string >,
+	themeSlugs: Array< string >
+): ThankYouSectionProps {
 	const translate = useTranslate();
-	const siteId = useSelector( getSelectedSiteId );
 	const siteSlug = useSelector( getSelectedSiteSlug );
+	const [ hasPlugins, hasThemes ] = [ pluginSlugs, themeSlugs ].map(
```

Comment:
Nit: I think a two-liner here would be more obvious. What do you think? 
```
const hasPlugins = pluginSlugs.length !== 0;
const hasThemes = themeSlugs.length !== 0;
```

Reply:
I do agree with two variables, but I think it will be easier in the future to just add slug variables to this array:

```js
const [ hasPlugins, hasThemes, hasDomains, hasEmail ] = [ pluginSlugs, themeSlugs, domainsSlugs, emailSlugs ].map(
```